src/cli/sync.ts

```diff
@@ -38,23 +39,16 @@ function watcher(dir: string) {
     persistent: true,
   });
 
-  watcher.on('add', async (filePath, stats) => {
-    const relativePath = path.relative(cwd(), filePath);
-    const newAsset = await createAsset(relativePath, {
-      size: stats?.size,
-    });
+  watcher.on('add', async (filePath) => {
+    const newAsset = await createAsset(filePath);
 
     if (newAsset) {
       log.add(`New file found: ${filePath}`);
-      return callHandler('local.video.added', newAsset, getCallHandlerConfig());
+      return callHandler('local.video.added', newAsset, await getVideoConfig());
```

Comment:
I'm not petty enough to demand this change, but something makes me twitch a little when I see `await` used inline like this. Would it help readability if this was just a variable declaration before used here?